update col types

kgaonkar6 opened the pull request:
<!--Hi there, thanks for your contribution! Please take a moment to fill out this template to facilitate the review of your pull request.-->

### Purpose/implementation Section

#### What scientific question is your analysis addressing?
v19 maf files have updated columns https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/990#issuecomment-826879406.


#### What was your approach?
I updated the maf_coltypes.RDS to include the HotSpotAllele and remove column names not part of the maf formats. Since the individual callers also have some unique columns I'm filtering the individual caller files to the columns in maf_coltypes.RDS.

#### What GitHub issue does your pull request address?
#990 


### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

#### Which areas should receive a particularly close look?
NA


#### Is there anything that you want to discuss further?
NA


#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes


### Results

#### What types of results are included (e.g., table, figure)?
No results ( intermediate files are placed in scratch) the output will be created in #956 


#### What is your summary of the results?
NA


#### Reproducibility Checklist

<!-- Check all those that apply or remove this section if it is not applicable.-->

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

<!-- Please review https://github.com/AlexsLemonade/OpenPBTA-analysis#documenting-your-analysis -->

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

<!-- IF YOUR PULL REQUEST IS A DATA RELEASE, PLEASE REMOVE THE [HTML COMMENT TAG](https://html.com/tags/comment-tag/) FROM THE SECTION BELOW AND COMPLETE THE CHECKLIST-->

<!-- 
### Data Release Checklist
- [ ] Is the table in doc/data-file-descriptions.md up to date?
- [ ] Is doc/data-format.md up to date?
- [ ] Is doc/release-notes.md up to date?
- [ ] Is download-data.sh up to date?
- [ ] Was download-data.sh tested and did it complete without error?
-->

jashapiro reviewed (commented):
I think it would be worth including a very small script in `utils`  that generates the coltypes RDS in addition to the RDS itself. This will make the change here more transparent, and easier for future users to adapt if necessary.

kgaonkar6 commented:
Added a script to create maf_coltypes.RDS. I didn't number this script since we already have a 00-* script. I've added it to the run script to add it as a step in the analysis.
